index.js

```diff
@@ -88,13 +88,13 @@ module.exports = function (opts) {
 
   async function parseBody(ctx) {
     if (enableJson && ((detectJSON && detectJSON(ctx)) || ctx.request.is(jsonTypes))) {
-      return await parse.json(ctx, jsonOpts);
+      return await parse.json(ctx, Object.assign({}, jsonOpts));
```

Comment:
co-body will change "length" field which is different between requests, so length and request will mismatch.